Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -2204,14 +2204,16 @@ def fetch_notable_users(client: Client, args: dict[str, str], last_run_obj: dict
         last_run_time = datetime.fromisoformat(last_run_notable_users).astimezone(timezone.utc)
         difference = current_time - last_run_time
         difference_minutes = difference.total_seconds() / 60
-
         fetch_interval = arg_to_number(args.get("notable_users_fetch_interval")) or 60
+        if fetch_interval % 60 != 0:
```

Comment:
you should do this validation in the test module. and here just to change it to be whole hours if its not.